bin/trackers/Tracker_Yara.py

```diff
@@ -96,10 +96,26 @@ def yara_rules_match(self, data):
             print(f'Send Mail {mail_subject}')
             NotificationHelper.sendEmailNotification(mail, mail_subject, mail_body)
 
+        # Webhook
+        webhook_to_post = Term.get_term_webhook(tracker_uuid)
+        if webhook_to_post:
+            json_request = {"trackerId": tracker_uuid,
+                            "itemId": item_id,
+                            "itemURL": self.full_item_url + item_id,
+                            "dataRule": data["rule"],
+                            "itemSource": item_source,
+                            "itemDate": item_date,
+                            "tags": tags_to_add,
+                            "emailNotification": f'{mail_to_notify}',
+                            "trackerType": "yara"
+                            }
+            response = requests.post(webhook_to_post, json=json_request)
+            if response.status_code >= 400:
+                raise Exception(f"Webhook request failed for {webhook_to_post}\nReason: {response.reason}")
```

Comment:
I don't think we should stop the module if we can't reach the webhook.
Can you replace this exception by an error message ?

You can use this function:
```python
self.redis_logger.error(error_message)
```